site/pages/create-disruption.tsx

```diff
@@ -1,9 +1,142 @@
-const CreateDisruption = () => {
-    return <></>;
+import React, { ReactElement } from "react";
+import DateSelector from "components/DateSelector";
+import TimeSelector from "components/TimeSelector";
+import { BaseLayout } from "components/layout/Layout"
+import { DisruptionValidity } from "interfaces";
+
+const title = 'Create Disruptions';
+const description = 'Create Disruptions page for the Create Transport Disruptions Service';
+
+interface CreateDisruptionProps {
+    inputs: DisruptionValidity
+}
+
+const CreateDisruption = ({ inputs }: CreateDisruptionProps): ReactElement => {
+    return (
+        <BaseLayout title={title} description={description}>
+            <>
+                <div className="govuk-form-group">
+                    <h1 class="govuk-heading-l">
+                        When is the disruption?
+                    </h1>
+                    <fieldset className="govuk-fieldset" role="group" aria-describedby="start-date-hint">
+                        <legend className="govuk-fieldset__legend govuk-!-padding-top-2">
+                            What is the start date?
```

Comment:
In the designs, the questions are bold so might be worth wrapping them in a `<h3 className="govuk-heading-s govuk-!-margin-bottom-0">`

Reply:
done